src/presets/eu/src/eu-default.scss

```diff
@@ -175,6 +175,12 @@
 .ecl blockquote p:is([class*='ecl-u-']) {
   @extend %ecl-blockquote__citation;
 
+  margin-top: map.get(var.$blockquote, 'author', 'margin');
```

Comment:
juest seeing that this is 1.5rem in EU, isn't it too much? maybe it would be better to introduce a new variable or just use something like 0.5rem in both themes, no..?

Reply:
If I'm not wrong, it is already var(--s-m), 1rem in both themes.
Sure, we could set a fixed value, but I guess that 1rem is fine here